src/hocon.erl

```diff
@@ -107,7 +107,7 @@ parse(Tokens) ->
 
 -spec(transform(config(), ctx()) -> {ok, config()}).
 transform(Config, Ctx) ->
-    try include(substitute(Config), Ctx) of
+    try include(Config, Ctx) of
```

Comment:
missing substitute ?

Reply:
substitution is now done by resolve/1

Reply:
which means that substitution occurs before the kv-list is converted into map.